test/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/ApiProjectGenerator.java

```diff
@@ -38,15 +38,17 @@
     private static final String docs = "Docs";
     private static final String endpointCertificates = "Endpoint-certificates";
     private static final String openAPIFile = "swagger.yaml";
+    private static final String apiYamlFile = "api.yaml";
+
     /**
      * Create apictl project zip file.
      *
      * @param apiYamlPath openapi file path.
      *
      * @throws MicroGWTestException
      */
-    public static String createApictlProjZip(String apiYamlPath) throws MicroGWTestException {
-        return createApictlProjZip(apiYamlPath, null);
+    public static String createApictlProjZip(String apiYamlPath, String swaggerPath) throws MicroGWTestException {
```

Comment:
for variable names also use term `openapi` and avoid `swagger` where possible

Reply:
Will do. Adding a todo locally, to include in the next PR of the same feature